### DISCUSSION TITLE
Using the browser extension on Firefox forks?

### DISCUSSION THREAD

**Discussion by 8L4YD** [keepassxreboot/keepassxc#11198] [Q&A] [answered]
Hi all,
I was just wondering if it was possible to set up a custom browser config location to get the Browser plugin to work with Firefox derivatives such as Floorp or Zen? I'd love to know how to get it working (where the config files go, etc.) for flatpaks or native apps, thanks! I'm on Linux (Fedora 40) with KeePassXC installed through dnf (not flatpak) if any of that is relevant :)

**Comment by alghostzx** [comment]
Was searching for a way to do it too, specifically for zen

**Reply by varjolintu** [reply, marked as answer]
You can use the Custom Browser (KeePassXC settings -> Browser Integration -> Advanced) feature to define a custom folder for the native messaging. KeePassXC will then create the script needed for connecting the browser to the application.

The path needed depends on the browser's configuration path.

**Comment by processor286** [comment]
What can you do if you use multiple forks which don't share a profile folder? E.g. Zen and Libre Wolf?

**Reply by varjolintu** [reply]
You will have to copy or create a symlink of the JSON file to a correct path. The Custom Browser option supports only a single entry for now.
